app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/AddressController.scala

```diff
@@ -37,14 +37,14 @@ import shapeless.{Lens, lens}
 
 @Singleton
 class AddressController @Inject()(
-  val authenticatedAction: AuthenticatedAction,
-  val subscriptionDetailsAction: SubscriptionReadyAction,
-  cc: MessagesControllerComponents,
-  errorHandler: ErrorHandler,
-  addressLookupService: AddressLookupService,
-  sessionStore: SessionStore,
-  enterPostcodePage: views.html.subscription.enter_postcode,
-  selectAddressPage: views.html.subscription.select_address
+                                   val authenticatedAction: AuthenticatedAction,
+                                   val subscriptionDetailsAction: SubscriptionReadyAction,
+                                   cc: MessagesControllerComponents,
+                                   errorHandler: ErrorHandler,
+                                   addressLookupService: AddressLookupService,
+                                   sessionStore: SessionStore,
+                                   enterPostcodePage: views.html.subscription.enter_postcode,
+                                   selectAddressPage: views.html.subscription.select_address
 )(implicit viewConfig: ViewConfig, ec: ExecutionContext)
```

Comment:
Why add the extra white space?

Reply:
I think my formatter automatically did something funny here - not sure how since we're all using the same scalafmt config file in theory